app/views/articles/_actions.html.erb

```diff
@@ -68,44 +68,44 @@
             <span id="article-copy-link-announcer" role="alert" hidden>Copied to Clipboard</span>
           </clipboard-copy>
         </div>
-        <web-share-wrapper shareurl="https://dev.to<%= @article.path %>" sharetext="<%= @article.title %>" template="web-share-button">
+        <web-share-wrapper shareurl="<%= article_url(@article) %>" sharetext="<%= @article.title %>" template="web-share-button">
           <div class="dropdown-link-row">
             <a
               target="_blank"
               rel="noopener"
-              href='https://twitter.com/intent/tweet?text="<%= u @article.title.strip %>" by <%= @article.user.twitter_username ? "@" + @article.user.twitter_username : @article.user.name %> %23DEVcommunity https://dev.to<%= @article.path %>'>
+              href='https://twitter.com/intent/tweet?text="<%= u @article.title.strip %>" by <%= @article.user.twitter_username ? "@" + @article.user.twitter_username : @article.user.name %> %23DEVcommunity <%= article_url(@article) %>'>
               Share to Twitter
             </a>
           </div>
           <div class="dropdown-link-row">
             <a
               target="_blank"
               rel="noopener"
-              href="https://www.linkedin.com/shareArticle?mini=true&url=https://dev.to<%= @article.path %>&title=<%= @article.title %>&summary=<%= @article.description %>&source=dev.to">
+              href="https://www.linkedin.com/shareArticle?mini=true&url=<%= article_url(@article) %>&title=<%= @article.title %>&summary=<%= @article.description %>&source=<%= community_name %>">
```

Comment:
Need a sanity check on this:
`&source=dev.to` to `&source=<%= community_name %>`. This changes the source from dev.to to DEV. Will it affect analytics

Reply:
It seems like we use the outdated way to share. I was only able to find [these docs](https://docs.microsoft.com/en-us/linkedin/consumer/integrations/self-serve/share-on-linkedin?context=linkedin/consumer/context) . I also tried the links themselves with sources `dev.to` and `DEV`, and the result was the same.

Reply:
Thanks Anna - yeah I did the same and I couldn't find any more information that you did:( At the same time, didn't want to break anything that wan't obvious to the eye

Reply:
I think for this pr we can keep it as is (with your change).